src/z3c/rml/rml.dtd

```diff
@@ -1918,7 +1921,7 @@
 <!ATTLIST li bulletFormat CDATA #IMPLIED>
 <!ATTLIST li bulletType (I | i | 1 | A | a | l | L | O | o | R | r | bulletchar | bullet | circle | square | disc | diamond | rarrowhead | diamondwx | sparkle | squarelrs | blackstar) #IMPLIED>
 <!ATTLIST li style CDATA #IMPLIED>
-<!ATTLIST li value (bulletchar | bullet | circle | square | disc | diamond | rarrowhead | diamondwx | sparkle | squarelrs | blackstar) #IMPLIED>
```

Comment:
i assume originally this was just a goof?

Reply:
Oh, yes, that one seems to dissappear automatically, it's weird, I forgot to check before I pushed last time, clearly.

Reply:
Yeah, I think so, because the other `li` entries don't include it, and I don't think it's correct.

In any case rml.dtd is automatically generated, so if I don't remove it now, it will just go away next time.